opentaskspal/src/main/java/org/dmfs/opentaskspal/tasks/DateTimeListTaskData.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2018 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.opentaskspal.tasks;
+
+import android.content.ContentProviderOperation;
+import android.support.annotation.NonNull;
+import android.text.TextUtils;
+
+import org.dmfs.android.contentpal.RowData;
+import org.dmfs.android.contentpal.TransactionContext;
+import org.dmfs.jems.iterable.decorators.Mapped;
+import org.dmfs.rfc5545.DateTime;
+import org.dmfs.tasks.contract.TaskContract;
+
+
+/**
+ * {@link RowData} for a field of {@link DateTime}s.
+ *
+ * @author Marten Gajda
+ */
+public final class DateTimeListTaskData implements RowData<TaskContract.Tasks>
```

Comment:
Making it package-private perhaps?

Reply:
I chose to generalize it and moved it to its own package. It might be useful in other places (although I don't see any atm). Check again.